values/vault-operator/vault-operator.gotmpl

```diff
@@ -5,3 +5,14 @@ global:
   imagePullSecrets:
     - otomi-pullsecret-global
 {{- end }}
+
+podSecurityContext:
+  runAsNonRoot: true
```

Comment:
`runAsNonRoot` is redundant when `runAsUser` is specified other than `0`

Reply:
done